### PR TITLE
Provide FAQ topics

### PR DESCRIPTION
**Pull request by tomkerkhove**
Provide more FAQ entries:
- What is the value on the HPA that I'm seeing?
- Update HTTP FAQ to fix dead line and point to our experimental add-on scaler
- Introduce best practices
  - Using multiple triggers
  - Mixing KEDA with HPA is not a good bet

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)
- [x] A PR is opened to update the documentation on [our docs repo](https://github.com/kedacore/keda-docs)

### CODE REVIEW

File: data/faq20.toml [hunk 1]
```diff
@@ -32,6 +35,39 @@ q = "Is short polling intervals a problem?"
...
+[[qna]]
+q = "Using multiple triggers for the same scale target"
+a = """
+KEDA allows you to use multiple triggers as part of the same `ScaledObject` or `ScaledJob`.
+
+By doing this, your autoscaling becomes better:
+- All your autoscaling rules are in one place
+- You will not have multiple `ScaledObject`'s or `ScaledJob`'s interferring with each other
+
+KEDA will start scaling as soon as when one of the triggers meets the criteria.
```
Comment (zroubalik) [inline data/faq20.toml:47]: I'd add a note that the maximum number is used for the target replica count calculation

**Review by zroubalik — COMMENTED** [review]
Looking good, minor nit on multiple triggers

File: data/faq20.toml (hunk 1 above)
Comment (tomkerkhove) [reply data/faq20.toml:47]: You mean the maximum number of what? The configured value for that trigger or?

Comment (zroubalik) [reply data/faq20.toml:47]: I meant max number acrross multiple triggers. Let' say you have mutliple triggers and each reports different metrics eg: trigger1: 5, trigger2: 10, trigger3: 0. Then HPA will  do the calculation on each of these metrics and will select the largest number for desired replica count.

Comment (tomkerkhove) [reply data/faq20.toml:47]: What about this?
```suggestion
KEDA will start scaling as soon as when one of the triggers meets the criteria. Horizontal Pod Autoscaler (HPA) will calculate metrics for every scaler and use the highest desired replica count to scale the workload to.
```

Comment (zroubalik) [reply data/faq20.toml:47]: Perfect :)

**Review by zroubalik — APPROVED** [review]

**Pull request merged**